src/odemis/util/angleres.py

```diff
@@ -514,10 +518,26 @@ def Rectangular2Polar(data, output_size, colormap=None):
     # Note: Everything outside of mask is 0
     mask = _CreateMirrorMaskRectangular(data, hole=True)
 
-    # Get the upper/lower intensity limit, by checking for a fixed amount of outliers.
     # In case of NaNs in projection, remove NaNs for calculating histogram limits, but keep for plotting.
     mask_outliers = mask & ~numpy.isnan(data)  # create new mask, which masks NaNs in data
-    lim1, lim2 = img.getOutliers(data[mask_outliers], outliers=1/256)  # 1/256 seems to work well for most datasets
+
+    # define the limits for plotting
+    if data.metadata[MD_POL_MODE] in [MD_POL_UP, MD_POL_DOP, MD_POL_DOLP]:
+        lim1, lim2 = 0, 1
+    # choose limits symmetrically for the following pol pos around 0
+    # DOCP: directly indicates whether data more rhc or lhc polarized
```

Comment:
Let's try to make the comments clearer. Please, start the sentences with an uppercase, and end them with a period.
I'd suggest to write:
```
    # For the following pol pos, choose the limits symmetrically around 0.
    # DOCP: highlight whether the data is more RHC- or LHC-polarized.
```